CONTRIBUTING.md

```diff
@@ -45,6 +45,14 @@ To run code linter, use:
 npm run lint
 ```
 
+## API guidelines
+
+When authoring new API methods, consider the following:
+- expose as little information as needed. When in doubt, don’t expose new information
+- methods are used in favor of getters/setters
+  - the only exception is namespaces, e.g. `page.keyboard` and `page.coverage`
+- all string literals must be small case. This includes event names and option values
```

Comment:
There have been questions in slack about adding new sugar apis (`page.exist()`). It might be nice to call out a bullet that discusses our philosophy on adding new apis.

Reply:
@ebidel good point; added a note